tracking-exposed/web-extension
# address the issue triggered when a reviewer test the extension

This should allow us to:
- generate distribution package who do not depends on git
- handle the lack of git, if any 

to test this script, you should:
- copy web-extension in another directory 
- enter the target directory and remove `.git`

This should set the environment in the equivalent of a reviewer who simply download the .zip from github
